src/test/codewhisperer/commands/transformByQ.test.ts

```diff
@@ -117,6 +108,7 @@ describe('transformByQ', function () {
                 message: 'No Java projects found',
             }
         )
+        fs.rmSync(dummyPomPath)
```

Comment:
no needed if `createTestWorkspaceFolder` is used, it cleans itself up after test suite is finished. https://github.com/aws/aws-toolkit-vscode/blob/1e7da02a5bea08de4f3bd97e7b2f7b938661a5a5/src/test/testUtil.ts#L62